testing/esp/wudetails/wutest.py

```diff
@@ -0,0 +1,549 @@
+#! /usr/bin/python3
+################################################################################
+#    HPCC SYSTEMS software Copyright (C) 2018 HPCC Systems®.
+#
+#    Licensed under the Apache License, Version 2.0 (the "License");
+#    you may not use this file except in compliance with the License.
+#    You may obtain a copy of the License at
+#
+#       http://www.apache.org/licenses/LICENSE-2.0
+#
+#    Unless required by applicable law or agreed to in writing, software
+#    distributed under the License is distributed on an "AS IS" BASIS,
+#    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    See the License for the specific language governing permissions and
+#    limitations under the License.
+################################################################################
+
+import argparse
+import filecmp
+import logging
+import logging.config
+import datetime
+import requests.packages.urllib3
+from requests import Session
+from requests.auth import HTTPBasicAuth
+from requests.auth import HTTPDigestAuth
+from zeep import Client 
+from zeep.transports import Transport
+from zeep.cache import SqliteCache
+from collections import namedtuple
+from pathlib import Path
+
+print('WUDetails Regression (wutest.py)')
+print('--------------------------------')
+print('')
+
+argumentparser = argparse.ArgumentParser()
+argumentparser.add_argument('-o', '--outdir', help='Results directory', default='results', metavar='dir')
+argumentparser.add_argument('-d', '--debug', help='Enable debug', action='store_true', default=False)
+argumentparser.add_argument('--logresp', help='Log wudetails responses (in results directory)', action='store_true', default=False)
+argumentparser.add_argument('--logreq', help='Log wudetails requests (in results directory)', action='store_true', default=False)
+argumentparser.add_argument('-n', '--lastndays', help="Use workunits from last 'n' days", type=int, default=1, metavar='days')
+argumentparser.add_argument('-t', '--testcase', help="Execute give testcase", type=int, default=0, metavar='number')
```

Comment:
did you mean "given"?

Reply:
I've actually removed this functionality.  The functionality makes less sense given that the executing full execution of all the test cases is quick... updating